services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -144,6 +144,59 @@ class GitLabProjectVersioningTest
       }
     }
 
+    "updateRepository" should {
+
+      def request(postProject: PostProject): Future[Response[GitLabRepositoryResponse]] =
+        mockHttpClient.put[GitLabRepositoryResponse, PostProject](
+          url = s"${gitLabConfig.url}projects/${updatingProject.repositoryId.value}",
+          headers = gitLabConfig.token,
+          payload = postProject
+        )
+
+      "throw new VersioningException for inactive project" taggedAs Service in {
```

Comment:
```suggestion
      "fail with VersioningException for inactive project" taggedAs Service in {
```
Throwing an exception and failing with some result are two different things

Reply:
Ok, thanks. Now i see it :)
I will fix it.